src/pages/LandingPage/LandingPage.tsx

```diff
@@ -23,10 +22,10 @@ const LandingPage = () => {
             <NimbusLogo />
           </XStack>
           <YStack style={{ width: '100%', margin: '30vh 0 4vh' }}>
-            <Title color="$textPrimary">
+            <Text size={27} weight={'semibold'}>
               Light and performant clients, for all Ethereum validators.
-            </Title>
-            <Text size={15} weight="regular" color="$textPrimary">
+            </Text>
+            <Text size={15} weight="regular">
```

Comment:
Likewise here we might consider a token for the font size.

Reply:
Text component is expecting number